### DISCUSSION TITLE
2005/05/10/477

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#25] [Announcements]
# 2005/05/10/477

我今天跑到那几个 MSN Space 开发人员的 blog 上一看，好嘛，全是中国人在留言。

https://blog.qizhen.xyz/2005/05/10/477

**Comment by ruanqizhen** [comment]
lilred
2005年05月12日 01:19 编辑
Hi Qizhen… Thanks for visiting Redvelvet\’s Underground & for the nice comment… I wish I could read your space but there is a language barrier :( Please come back & visit my space anytime… Take care… CIAO!!!

回复
Linda
2005年05月11日 23:54 编辑
^—-^A really helpful page you created! Keep updating lar!

回复
小＆U
2005年05月11日 19:48 编辑
呵呵！拽~~~

回复
帅猪
2005年05月11日 17:08 编辑
支持！！

回复
shelly
2005年05月11日 17:02 编辑
在我的MSN SPACES 里发现你，不认识你，但猜是我妹在英国的朋友，对吗？很高兴认识你，要照顾一下KAREN啊，谢谢你光顾我刚刚建的SPACES。。。

回复
pup
2005年05月11日 14:11 编辑
hehe, 音乐不错，很喜欢:)

回复
Unknown
2005年05月11日 11:42 编辑
和和~~~我也来你这里溜达一圈~~~噶~~

回复
京
2005年05月11日 10:04 编辑
呵呵，来参观一下，顺便踩踩

回复
Bonny
2005年05月11日 09:45 编辑
谢谢踩我的小窝。。。回踩。。。 ：）

回复
Unknown
2005年05月11日 04:10 编辑
花花花吗?

回复
Jocelyn
2005年05月11日 01:39 编辑
我踩，看贴，必回，踩啊．．踩

回复
